src/components/WhatIsOpenChoreo/styles.module.css

```
@@ -2,67 +2,645 @@
 }
 
 .featureCard {
-  height: 100%;
-  padding: 1rem;
-  border-radius: 16px;
-  background: var(--ifm-background-surface-color);
+  --feature-accent: var(--ifm-color-primary);
+  --feature-accent-soft: rgba(24, 119, 242, 0.14);
+  --feature-accent-border: rgba(24, 119, 242, 0.18);
+  --feature-glow: rgba(24, 119, 242, 0.18);
+  position: relative;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)